osu.Game/Online/Spectator/SpectatorClient.cs

```diff
@@ -279,7 +279,7 @@ private void purgePendingFrames()
 
             totalBundledFrames += frames.Length;
 
-            Console.WriteLine($"Purging {pendingFrames.Count} frames (total {totalBundledFrames})");
+            Logger.Log($"Purging {pendingFrames.Count} frames (total {totalBundledFrames})");
```

Comment:
This can probably be removed in fact.. weird to have it outputting to the runtime logger as is...